lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -53,21 +53,25 @@
   version:
     description:
       - HSRP version.
-    default: 2
+    default: 1
     choices: ['1','2']
   priority:
     description:
-      - HSRP priority.
+      - HSRP priority or keyword 'default'.
   preempt:
     description:
       - Enable/Disable preempt.
     choices: ['enabled', 'disabled']
   vip:
     description:
-      - HSRP virtual IP address.
+      - HSRP virtual IP address or keyword 'default'
   auth_string:
     description:
-      - Authentication string.
+      - Authentication string. If this needs to be hidden(for md5 type), the string
+        should be 7 followed by the key string. Otherwise, it can be 0 followed by
+        key string or just key string (for backward compatibility). For text type,
+        this should be just be a key string. if this is 'default', authentication
+        is removed.
```

Comment:
Can you please add example for md5 authentication with '7' or '0' auth_string in the Example section?
other auth encryption can be confused as new parameter.

Reply:
Added examples, thanks for pointing this out.